include/xtensor/xmath.hpp

```diff
@@ -906,7 +912,7 @@ XTENSOR_INT_SPECIALIZATION_IMPL(FUNC_NAME, RETURN_VAL, unsigned long long);
      *     auto fnct = [](auto x) -> decltype(x * x) {
      *         return x * x;
      *     };
-     *     return make_lambda_xfunction(std::move(fnct), std::forward<E1>(e1));
+     *     return ufuncify(std::move(fnct), std::forward<E1>(e1));
```

Comment:
`xfunctorize`? ;)

Reply:
`make_lambda_xfunction` is ok, I agree `ufuncify` is not really meaningful for C++ developers.

Reply:
reverted.